api/app/reviews/tests.py

```diff
@@ -1285,3 +1288,246 @@ def test_total_number_of_pages_when_zero(self):
 
         self.assertEqual(data['total_pages'], 0)
     
+
+class ReferenceReviewRequest(ApiTestCase):
+    def static_seed_data(self):
+        # User, country and organisation is set up by ApiTestCase
+        self.first_user = self.add_user('firstuser@mail.com', 'First', 'User', 'Mx')
+
+        event = self.add_event()
+
+        reviewer1 = AppUser('r1@r.com', 'reviewer', '1', 'Mr', password='abc', organisation_id=1, )
+        candidate1 = AppUser('c1@c.com', 'candidate', '1', 'Mr', password='abc', organisation_id=1, )
+        system_admin = AppUser('sa@sa.com', 'system_admin', '1', 'Ms', password='abc', organisation_id=1, is_admin=True)
+        event_admin = AppUser('ea@ea.com', 'event_admin', '1', 'Ms', password='abc', organisation_id=1)
+        users = [reviewer1, candidate1, system_admin, event_admin]
+        for user in users:
+            user.verify()
+        db.session.add_all(users)
+        db.session.commit()
+
+        event_roles = [
+            EventRole('admin', 4, 1),
+            EventRole('reviewer', 1, 1)
+        ]
+        db.session.add_all(event_roles)
+        db.session.commit()
+
+        application_form = [
+            self.create_application_form(1, True, False),
+        ]
+        db.session.add_all(application_form)
+        db.session.commit()
+
+        closed_review = ReviewForm(1, datetime(2018, 4, 30))
+        closed_review.close()
+        review_forms = [
+            ReviewForm(1, datetime(2019, 4, 30)),
+            closed_review
+        ]
+        db.session.add_all(review_forms)
+        db.session.commit()
+
+        self.test_response = Response(  # Nominating other
+            1, self.first_user.id)
+        self.test_response.submit()
+
+        db.session.add(self.test_response)
+        db.session.commit()
+
+        response_review = ResponseReviewer(self.test_response.id, self.first_user.id)

```

Comment:
Please add a new method to testing.py called add_response_reviewer() which does this. 
We prefer to centralise the functions that create entities for tests in one place so that if someone adds a constructor parameter to an entity in future then we can just update it in one place in testing.py instead of in lots of different test files :)

Reply:
@avishkar58, I see someone already added [add_response_reviewer](https://github.com/deep-learning-indaba/Baobab/blob/a5b4fa192dfe86869b7df01bcb5cbb8914b69b24/api/app/utils/testing.py#L355).

I've left a comment in my new commit. I'm struggling with this and the self.add_response :)